src/server/conn.rs

```diff
@@ -24,15 +24,16 @@ use super::{Result, ConnData};
 use super::server::Server;
 use util::codec::rpc;
 use super::transport::RaftStoreRouter;
+use super::resolve::StoreAddrResolver;
 
 pub struct Conn {
     pub sock: TcpStream,
     pub token: Token,
     pub interest: EventSet,
 
-    // peer_addr is for remote peer address, we only set this
-    // when we connect to the remote peer.
-    pub peer_addr: Option<String>,
+    // store id is for remote store, we only set this
+    // when we connect to the remote store.
+    pub store_id: Option<u64>,
```

Comment:
This is only for connecting remote store proactively,  I think Option is ok here.